$DragonFly: src/gnu/usr.bin/gdb/gdb/patches/Attic/i386fbsd-tdep.c.patch,v 1.1 2005/05/07 19:48:16 corecode Exp $
Index: i386fbsd-tdep.c
===================================================================
RCS file: /space/cvs/dragonfly/src/contrib/gdb-6.2.1/gdb/i386fbsd-tdep.c,v
retrieving revision 1.1
diff -u -r1.1 i386fbsd-tdep.c
--- i386fbsd-tdep.c	24 Oct 2004 19:56:25 -0000	1.1
+++ i386fbsd-tdep.c	7 May 2005 18:56:17 -0000
@@ -45,7 +45,7 @@
 CORE_ADDR i386fbsd_sigtramp_end_addr = 0xbfbfdff0;
 
 /* From <machine/signal.h>.  */
-static int i386fbsd_sc_reg_offset[] =
+int i386fbsd_sc_reg_offset[] =
 {
   8 + 14 * 4,			/* %eax */
   8 + 13 * 4,			/* %ecx */
